src/pages/PublicAppointments/auth/PatientLogin.tsx

```diff
@@ -6,6 +6,7 @@ import { useState } from "react";
 import { useForm } from "react-hook-form";
 import { useTranslation } from "react-i18next";
 import { isValidPhoneNumber } from "react-phone-number-input";
+import { ToastContainer, toast } from "react-toastify";
```

Comment:
why do we need an additional package for toast notifications?!!

we already have one; just reuse it